app/models/file_resource.rb

```diff
@@ -19,6 +19,8 @@ class FileResource < ApplicationRecord
           required: false,
           dependent: :destroy
 
+  has_one :accessibility_check_result, dependent: :destroy
```

Comment:
I put a `required: false` in here, but I'm realizing that it's not needed.  So if you encounter this as a merge conflict, it can be removed.